.github/workflows/ci.yml

```diff
@@ -101,22 +101,26 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-${{ matrix.compiler }}-${{matrix.build_type}}-${{matrix.generator}}-${{matrix.developer_mode}}
 
+      - uses: actions/setup-python@v2
+
+      - run: pip3 install cmake ninja conan gcovr
+
       - name: Setup Cpp
         uses: aminya/setup-cpp@v1
         with:
           compiler: ${{ matrix.compiler }}
           vcvarsall: ${{ contains(matrix.os, 'windows' )}}
 
-          cmake: true
-          ninja: true
-          conan: true
+          cmake: false
+          ninja: false
```

Comment:
Cmake and Ninja do not need python. Why should they be installed with pip3?

Reply:
Because it gives a simple consistent command that works on all 3 platforms that are supported on github. Since we are working specifically on a github workflow here.